clientUtils/owidTypes.ts

```diff
@@ -56,6 +56,8 @@ export interface RelatedChart {
 
 export type OwidVariableId = Integer // remove.
 
+export const BLOCK_WRAPPER_DATATYPE = "block-wrapper"
+
 export interface FormattedPost extends FullPost {
```

Comment:
Can't comment down there, but it probably makes sense to also make it
```ts
    tocHeadings: TocHeading[]
```
down below in `interface FormattedPost`?

Reply:
Agreed!